.github/workflows/ci.yml

```diff
@@ -47,7 +47,8 @@ jobs:
 
     - name: Install packages
       # `llvm-14-tools` is needed to install the `FileCheck` binary which is used for asm tests.
-      run: sudo apt-get install ninja-build ripgrep llvm-14-tools
+      run: |
+          sudo apt-get install ninja-build ripgrep llvm-14-tools libstdc++-11-dev
```

Comment:
Are you sure this change is needed?
I would expect it to be required in some other workflows if it was the case, but all the CI passed, so maybe it's not actually needed.

Reply:
not needed, I added tests failing here on txt file, would remove them. We could create a different issue for that linker error as well imo to add libstdc++.